src/components/Form/FormSchema.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import Form from 'react-jsonschema-form'
+
+import { useStepperDialogContext } from 'src/components/Hooks/useStepperDialogContext'
+import FileInputAdapter from 'src/components/ModelPages/widgets/FileInputAdapter'
+
+const ObjectFieldTemplate = props => {
+  return props.properties.map(element => {
+    return <React.Fragment key={element.name}>{element.content}</React.Fragment>
+  })
+}
+
+const widgets = { FileInputAdapter }
+const FormSchema = () => {
+  const { allCurrentPages, currentPageIndex } = useStepperDialogContext()
+  const [state, setState] = React.useState({})
+
+  const schema = React.useMemo(
+    () => allCurrentPages[currentPageIndex - 1].model.schema,
+    [allCurrentPages, currentPageIndex]
+  )
+  const uiSchema = React.useMemo(
+    () => allCurrentPages[currentPageIndex - 1].model.uiSchema,
+    [allCurrentPages, currentPageIndex]
+  )
+
+  const onChange = ({ formData }) => {
+    setState(prev => ({
+      ...prev,
+      ...formData
+    }))
+  }
+
+  return (
+    <Form
+      schema={schema}
+      uiSchema={uiSchema}
+      widgets={widgets}
+      onChange={onChange}
+      ObjectFieldTemplate={ObjectFieldTemplate}
+      formData={state}
+    >
```

Comment:
```<Form 
....
/> 
```
instead of 
```
<Form 
...>
<></>
</Form> 
``` ?

Reply:
Le bibliothèque rjsf va automatiquement générer un bouton `submit` si le composant `Form` n'a pas d'enfant
Une issue à ce sujet : https://github.com/commitd/components-rjsf/issues/1

Reply:
Peut-être à mettre en commentaire dans le code pour pas qu'on le supprimer dans 6 mois...